dp-hello-world-event/event/handler.go

```diff
@@ -19,6 +20,8 @@ func (h *HelloCalledHandler) Handle(ctx context.Context, event *HelloCalled) (er
 
 	//TODO Replace with actual event handler logic…
 	greeting := fmt.Sprintf("Hello, %s!", event.RecipientName)
+	_ = ioutil.WriteFile("/tmp/helloworld.txt", []byte(greeting), 0644)
```

Comment:
The 'Hello World' handlers serve as a sort of 'best practice' example so we should handle the errors that come back rather than ignore them. For this reason too, we should use a constant for the temporary file location.